verification/curator-service/api/src/model/user.ts

```diff
@@ -14,12 +14,17 @@ const userSchema = new mongoose.Schema({
         // supporting more identity platforms.
         required: 'User must be logged-in with Google',
     },
+    role: {
```

Comment:
Do we have a defined set of roles for which we can add an `enum` validator?

https://mongoosejs.com/docs/schematypes.html#string-validators

Reply:
yes let's use an enum please, we can safely assume we'll have "reader", "curator" and "admin" for now.